podman.spec.rpkg

```diff
@@ -207,6 +207,11 @@ PODMAN_VERSION=%{version} %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} ETCDI
         install.docker-docs \
         install.remote \
 
+# Add iptables modules for fedora36, https://github.com/containers/podman/issues/12661
+%if 0%{?fedora} >= 36
+    %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} ETCDIR=%{buildroot}%{_sysconfdir} install.modules-load \
```

Comment:
install.modules-load doesn't need the ETCDIR from what I can see. Best to remove `ETCDIR=%{buildroot}%{_sysconfdir}` .

Reply:
let me send a diff ..we can use a single make command...

Reply:
```
diff --git a/podman.spec.rpkg b/podman.spec.rpkg
index d02b7ea99..9925006ef 100644
--- a/podman.spec.rpkg
+++ b/podman.spec.rpkg
@@ -206,6 +206,9 @@ PODMAN_VERSION=%{version} %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} ETCDI
         install.docker \
         install.docker-docs \
         install.remote \
+%if 0%{?fedora} >= 36
+        install.modules-load
+%endif
```